src/utils/constants.js

```diff
@@ -29,4 +29,12 @@ export const SINGULARS = {
 	transactions: 'transaction',
 };
 
-export const CONFIG_VARIABLES = ['json', 'name', 'pretty', 'testnet'];
+export const CONFIG_VARIABLES = [
+	'json',
+	'name',
+	'pretty',
+	'liskJS.testnet',
+	'liskJS.ssl',
+	'liskJS.node',
+	'liskJS.port',
```

Comment:
How about renaming these `api.testnet` etc? It's for initializing specifically the API module of LiskJS.

Reply:
Yes, I like that.